subroutines.sh

```diff
@@ -249,3 +249,23 @@ delete_legacies() {
   fi
 
 }
+
+scan_bleln_macaddr() {
+  # copied from legacy "scan_bluetooth" function. To decide if still relevant
+  # note there is this PR https://github.com/tesla-local-control/tesla-local-control-addon/pull/32
+  # quite old, but has the principles for auto populating the BLE MAC Address with only the VIN
+  vin=$1
+  ble_ln=$(tesla_vin2ble_ln $vin)
+  log_debug "Calculated BLE Local Name:$ble_ln for vin:$vin"
+
+  log_info "Looking for vin:$vin in the BLE cache that matches ble_ln:$ble_ln"
+  if ! bluetoothctl --timeout 2 devices | grep $ble_ln; then
```

Comment:
will this print on stdout? could it be possible to explicitely log the MAC address in the final message?

Reply:
btlctl is colored output.. it shouldn't screw our logs at all, if needed we can strip off the colors with a sed.

For devices, the output will be like this something like:
`Device: ""MAC address"" ""BLE Local Name""`

For scan...

`[Bluetooth#} New MAC address BLE Local Name`
or
`[Bluetooth#} Del MAC address BLE Local Name`